frontend/src/app/i18n/locales/en/signin.json

```diff
@@ -0,0 +1,6 @@
+{
+    "signIn": "Sign In",
+    "email": "Email",
+    "password": "Password",
+    "incorrectEmail": "Incorrect email"
```

Comment:
i haven't looked further yet, but is it not missing something like `Login failed, incorrect password or email` ? That case can happen right?